backend/src/models/event.js

```diff
@@ -8,8 +7,8 @@ const Event = new mongoose.Schema({
   endTime: { type: String, required: true },
   location: { type: String, required: true },
   description: { type: String, required: true },
-  attendees: { type: [mongoose.Schema.Types.ObjectId], default: [] },
-  lateAttendees: { type: [mongoose.Schema.Types.ObjectId], default: [] },
+  attendees: { type: [String], default: [] },
```

Comment:
Does it just track by name then?

Reply:
maybe store this as `attendeeEmails` and `lateAttendeeEmails`

Reply:
yea it tracks by email

Reply:
ok